test/controllers/IfsControllerSpec.scala

```diff
@@ -26,43 +26,54 @@ import play.api.test.Helpers._
 import java.time.OffsetDateTime
 import scala.concurrent.Future
 
-class IfsControllerSpec extends SpecBase{
+class IfsControllerSpec extends SpecBase {
 
   private val controller: IfsController = app.injector.instanceOf[IfsController]
 
-  private def submitStoreInteraction(calculationId: String): Future[Result] = {
+  private val links = Some(
+    Seq(
+      IFRequestPayloadActionLinks("[Canllawiau ITSA, Arweiniad i Ffynonellau Incwm]", "[www.itsa/cym.gov.uk, www.itsa/incomesources.gov.uk]"),
+      IFRequestPayloadActionLinks("[Cyfrifo am Incwm]", "[www.itsa/incomecompliance.gov.uk]")
+    )
+  )
+
+  private def submitStoreInteraction(calculationId: String, links: Option[Seq[IFRequestPayloadActionLinks]]): Future[Result] = {
```

Comment:
This can have default value as none so you dont have to update it in all other places

Reply:
Updated it, Please see latest code.